bin/README.md

```diff
@@ -1,22 +1,25 @@
 ## Running the web tool on transitionmonitor's docker-container
 
-Import the base images with:
-
-```bash
-docker image import https://tool.transitionmonitor.com/downloads/2diirunner_1.3.0.tar.xz 2diirunner:1.3.0
-```
-
```

Comment:
This is no longer relevant. We now use images from dockerhub.

Reply:
👍